rest_framework-stubs/decorators.pyi

```diff
@@ -21,19 +21,40 @@ class MethodMapper(dict):
     def options(self, func: Callable) -> Callable: ...
     def trace(self, func: Callable) -> Callable: ...
 
+LOWER_CASE_HTTP_VERBS = List[
+    Union[
+        Literal["get"],
+        Literal["post"],
+        Literal["delete"],
+        Literal["put"],
+        Literal["patch"],
+        Literal["trace"],
+        Literal["options"],
+    ]
+]
+
+MIXED_CASE_HTTP_VERBS = List[
```

Comment:
Just to clarify this:
1. we use `MIXED_CASE_HTTP_VERBS` in case there's an existing public contant in the original package
2. we use `_MIXED_CASE_HTTP_VERBS` in case we need to declare our own stuff

Reply:
Kk, fixed